data/index.js

```diff
@@ -46,3 +48,55 @@ export function registerReducer( key, reducer ) {
 		getState,
 	};
 }
+
+/**
+ * Registers selectors for external usage.
+ *
+ * @param {string} reducerKey          Part of the state shape to register the
+ *                                     selectors for.
+ * @param {Object} registeredSelectors Selectors to register. Keys will be used
```

Comment:
In docs we have `selectors` provided as the 2nd param, maybe we should use the same name here, what do you think?

Reply:
Okey, Eslint shadowing again. Maybye `newSelectors` then :)